userland/examples/nrf51_aes/aes.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <tock.h>
+
+#define DRIVER_ECB 34
+#define DRIVER_CCM 35
+#define KEY        0
+#define ENC        1
+#define DEC        2 
```

Comment:
I think just have a `DATA` for the `allow` system call instead of a separate allow for `ENC` and `DEC`. Also, probably should prefix these with `AES_CMD_`, `AED_ALLOW_` etc so not to pollute the global namespace.